Client/Assets/NetError.css

```diff
@@ -27,16 +27,16 @@ h1 {
 }
 
 /* Add a set of stripes at the top of pages */
-#decoration {
-    background-image: repeating-linear-gradient(-65deg,      #eee,  #eee 10px,
-                                                       white 10px, white 20px,
-                                                                    #eee 20px);
-    position: fixed;
-    top: 0;
-    left: 0;
-    height: 32px;
-    width: 100%;
-}
+/*#decoration {*/
```

Comment:
Is this block commented to keep the information of the changes in the class? (instead of deleting it)

Reply:
That's correct, it is the first time we actually move Firefox CSS around, so I guess commenting or deleting both would be fine.